app/services/HttpIdentityService.scala

```diff
@@ -95,9 +94,19 @@ case class GuestRegistrationResponse(
     guestRegistrationRequest: GuestRegistrationResponse.GuestRegistrationRequest
 )
 
+case class UserIdWithOptionalToken(userId: String, token: Option[String])
+
+object UserIdWithOptionalToken {
+  def fromGuestRegistrationResponse(guestRegistrationResponse: GuestRegistrationResponse): UserIdWithOptionalToken =
```

Comment:
This isn't getting called. Is it needed?

Reply:
Good spot, I was intending on using it in `createUserIdFromEmailUser` in the `HttpIdentityService` class. Will put in the call now